kore/src/Log/Entry.hs

```diff
@@ -92,15 +88,14 @@ entryTypeText (SomeEntry entry) =
 data ActualEntry =
     ActualEntry
         { actualEntry :: !SomeEntry
-        , entryContext :: !(Seq SomeEntry)
+        , entryContext :: ![SomeEntry]
```

Comment:
Same question as above.

Reply:
The `ActualEntry` changes because the `LoggerEnv` needs to change, to avoid converting between types when constructing the `ActualEntry`.